job_spec/WATER_MAP_EQ.yml

```
@@ -0,0 +1,114 @@
+WATER_MAP_EQ:
+  required_parameters:
+    - granules
+  parameters:
+    granules:
+      default: '""'
+      api_schema:
+        type: array
+        minItems: 1
+        maxItems: 1
+        items:
+          anyOf:
+            - description: The name of the IW VV+VH Sentinel-1 GRDH granule to process
+              type: string
+              pattern: "^S1[AB]_IW_GRDH_1S[SD]V"
+              minLength: 67
+              maxLength: 67
+              example: S1A_IW_GRDH_1SDV_20210413T235641_20210413T235706_037439_0469D0_3F2B
+            - description: The name of the IW VV+VH Sentinel-1 SLC granule to process
+              type: string
+              pattern: "^S1[AB]_IW_SLC__1S[SD]V"
+              minLength: 67
+              maxLength: 67
+              example: S1A_IW_SLC__1SDV_20211110T234815_20211110T234842_040516_04CE0A_E717
+    bucket_prefix:
+      default:  '""'
+    resolution:
+      api_schema:
+        default:  30.0
+        description: Desired output pixel spacing in meters
+        type: number
+        enum:
+          - 30.0
+          - 20.0
```

```suggestion
          - 20.0
```
No need for 20m on this project